pulpcore.te

```diff
@@ -96,6 +97,9 @@ fs_tmpfs_filetrans(pulpcore_t, pulpcore_server_tmpfs_t, file )
 allow pulpcore_t pulpcore_server_tmpfs_t:file map;
 
 # interface calls
+kernel_read_all_proc(pulpcore_t)
+kernel_read_all_proc(pulpcore_server_t)
```

Comment:
I couldn't really figure out what needed this, but I can confirm I also saw AVCs for proc. For my own understanding: any idea how to figure out why this is needed?

Reply:
Some of the audit records:
```
type=AVC msg=audit(1603222196.793:7984): avc:  denied  { open } for  pid=14026 comm="gunicorn" path="/proc/stat" dev="proc" ino=4026532029 scontext=system_u:system_r:pulpcore_server_t:s0 tcontext=system_u:object_r:proc_t:s0 tclass=file permissive=1
type=AVC msg=audit(1603292729.179:16858): avc:  denied  { getattr } for  pid=9265 comm="sh" path="/proc/meminfo" dev="proc" ino=4026532028 scontext=system_u:system_r:pulpcore_server_t:s0 tcontext=system_u:object_r:proc_t:s0 tclass=file permissive=1
```
Either it is required by the processes code or libraries they use.